packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/List.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Font } from '@samuelmeuli/font-manager';
+import { List, SearchFonts, FontLI, FontFamily } from './elements';
+
+function getFontId(fontFamily: string): string {
+  return fontFamily.replace(/\s+/g, '-').toLowerCase();
+}
+
+type Props = {
+  fonts: Font[];
+  onSelection: (e: any) => void;
+  activeFontFamily: string;
+  expanded: boolean;
+  suffix: string;
+};
+
+const FontList = ({
+  fonts,
+  onSelection,
+  activeFontFamily,
+  expanded,
+  suffix,
+}: Props) => {
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const updateSearch = e => setSearchTerm(e.target.value);
+  return (
+    <List expanded={expanded}>
+      <SearchFonts
+        type="text"
+        value={searchTerm}
+        onChange={updateSearch}
+        placeholder="Search Typefaces"
+      />
+      {fonts
+        .filter(f =>
+          f.family.toLowerCase().includes(searchTerm.trim().toLowerCase())
+        )
```

Comment:
i'd extract this filter up for readability, (plus you can memoize it based on the search term)

Reply:
nice!